modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/model/listener/UserExpandoColumnModelListener.java

```diff
@@ -191,6 +192,13 @@ else if (expandoColumn.getType() == ExpandoColumnConstants.DATE) {
 				locale -> Field.getSortableFieldName(encodedIndexedFieldName),
 				locale -> encodedIndexedFieldName);
 		}
+		else if ((expandoColumn.getType() == ExpandoColumnConstants.SHORT) ||
```

Comment:
Normally It's good to sort conditions alphabetically, if logic is not affected with the change

```java
		else if ((expandoColumn.getType() == ExpandoColumnConstants.INTEGER) ||
				 (expandoColumn.getType() == ExpandoColumnConstants.LONG) ||
				 (expandoColumn.getType() == ExpandoColumnConstants.SHORT)) {

			entityField = new IntegerEntityField(
				encodedName, locale -> encodedIndexedFieldName);
		}
```

Reply:
Addressed in #37.